Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:noble
+RUN apt-get update && apt-get install -y ca-certificates wget unzip git make && update-ca-certificates
+
+RUN mkdir -p ~/.eval-dev-quality
+
+# Install Maven
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz && \
+	tar -xf apache-maven-3.9.8-bin.tar.gz -C ~/.eval-dev-quality/  && \
+	rm apache-maven-3.9.8-bin.tar.gz
+ENV PATH="${PATH}:/root/.eval-dev-quality/apache-maven-3.9.8/bin"
+
+# Install Gradle
+RUN wget https://services.gradle.org/distributions/gradle-8.8-bin.zip && \
+	unzip gradle-8.8-bin.zip -d ~/.eval-dev-quality/ && \
+	rm gradle-8.8-bin.zip
+ENV PATH="${PATH}:/root/.eval-dev-quality/gradle-8.8/bin"
+
+# Install Java
+RUN wget https://download.oracle.com/java/22/latest/jdk-22_linux-x64_bin.tar.gz && \
+	tar -xf jdk-22_linux-x64_bin.tar.gz -C ~/.eval-dev-quality/ && \
+	rm jdk-22_linux-x64_bin.tar.gz
+ENV JAVA_HOME="/root/.eval-dev-quality/jdk-22.0.1"
+ENV PATH="${PATH}:${JAVA_HOME}/bin"
+
+# Install Go
+RUN wget https://go.dev/dl/go1.22.4.linux-amd64.tar.gz && \
```

Comment:
Please use 1:1 the same versions we use locally right now. So Go 1.21.5 here (for the others, i did not check) reason is that that is what we use in the CI, and we don't have "works here" problems.